src/lcals-kokkos/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+###############################################################################
+# Copyright (c) 2017-20, Lawrence Livermore National Security, LLC
```

Comment:
Please update the copyright dates on all the file. They should be '2017-22'.

Reply:
@rhornung67, @MrBurmark  -- I pushed the corrected copyright dates to my PR #256.  Please LMK if you anything else is needed before merging.

Reply:
@ajpowelsnl we need those changes to go into the PR branch in our repo that we made from your original PR on a forked branch, which will eventually be merged. Our complete CI checks can only be run on branches on our repo, which is required for merging. 

Do you have privileges to push those changes to the branch for PR https://github.com/LLNL/RAJAPerf/pull/267?

Reply:
If not, there is another way, which I will work on.